### PR TITLE
Jan. 2021 news

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-team  Please add your contribution from Dec. 2020.

(refs #11447)

### CODE REVIEW

**Comment by loganharbour** [comment]
December 2020?!

**Comment by aeslaughter** [comment]
Jan. 2021 news, Dec. 2020 content

**Comment by fdkong** [comment]
I resolved the conflict, but I need to force push. Anyone is against this action?

**Comment by loganharbour** [comment]
Go for it.

File: modules/doc/content/newsletter/2021_01.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ Failure analysis reports were added to the SQA documentation, see [framework_far
 
 Documentation for setting up MOOSE-based applications to leverage capabilities of MooseDocs
 for SQA was added: [extensions/sqa.md].
+
+# Strumpack Support
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2021_01.md:31]:
```suggestion
## Strumpack Support
```
@fdkong I would make the heading a little smaller to fit with the rest.

Comment (fdkong) [reply modules/doc/content/newsletter/2021_01.md]: Thanks!

**Review by loganharbour — APPROVED** [review]
Once https://civet.inl.gov/event/62263/ merges - let's get this in. I've pinged a few about contributions and it seems we have all of them.

**Pull request merged**